testdata/p4_16_samples_outputs/issue2267-first.p4

```diff
@@ -0,0 +1,16 @@
+#include <core.p4>
+
+control MyC(bit<8> t) {
+    table t {
+        key = {
+            t: exact @name("t") ;
+        }
+        actions = {
+            @defaultonly NoAction();
+        }
+        default_action = NoAction();
+    }
+    apply {
+    }
+}
+
```

Comment:
Consider changing this test case to

    control MyC(bit<8> t, bit<8> u) {
        table t {
            key = { u: exact @name("u") ; }
        }
        table u {
            key = { t: exact @name("t") ; }
        }
        apply {
            t.apply();
            u.apply();
        }
    }